Makefile

```diff
@@ -57,18 +57,19 @@ install-race:                   ## Install pmm-agent binary with race detector.
 
 TEST_PACKAGES ?= ./...
 TEST_FLAGS ?= -timeout=60s
+TEST_ENV_VARS ?= GODEBUG=x509ignoreCN=0
```

Comment:
That line makes all testing efforts meaningless – essentially, we are not testing with certificates that our users could use.

Please generate valid certificate with https://github.com/FiloSottile/mkcert (`mkcert -install; mkcert -cert-file=XXX -key-file=XXX localhost 127.0.0.1`) in Makefile and run that target on CI.

Reply:
regenerated certificates with mkcert

Reply:
`mkcert -install` should be called on CI, and certificates should be generated there too, not committed in the repo – it will not work otherwise. Please take a moment to read mkcert's README